pkgs/development/libraries/gstreamer/rs/default.nix

```diff
@@ -120,6 +121,11 @@ let
       "livesync"
       "fallbackswitch"
       "togglerecord"
+    ] ++ lib.optionals (!withGtkPlugins) [
+      "fallbackswitch"
+      "gtk4"
+      "livesync"
+      "togglerecord"
```

Comment:
Couldn't this be merged with the conditon above to `!gst-plugins-base.glEnabled || !withGtkPlugins` as they contain the same four entries.